src/libs/logos.js

```diff
@@ -63,3 +64,13 @@ export const topBarLogo = () => isTerra() ?
   brandLogoMaker(50, true)
 
 export const footerLogo = () => isTerra() ? terraLogoMaker(terraLogoWhite, { height: 40 }) : brandLogoMaker(40)
+
+export const versionTag = (children, styles) => b({
```

Comment:
As written, this isn't quite a component; it's a function that takes 2 arguments and returns a component. To be a component, it should take a `props` object (and should start with an uppercase letter):

```
export const VersionTag = ({styles, children}) => b({
  ...
```

And then `children` below should not be in `[]` because it should already be an array.

Alternatively, if you want to keep it as a function instead of a component, keep the name as is and change it back to `version`; `children` to me implies that it's a component.

Reply:
Good point, lets leave it as a function.